app/assets/stylesheets/basic.scss

```diff
@@ -0,0 +1,36 @@
+
+
+.background {
+  display: flex;
```

Comment:
Properties should be ordered background-color, -moz-background-size, -o-background-size, -webkit-background-size, background-size, display, flex-flow, height, overflow